assets/js/googlesitekit/datastore/site/index.test.js

```diff
@@ -58,4 +59,17 @@ describe( 'core/site store', () => {
 			expect( state ).toEqual( INITIAL_STATE );
 		} );
 	} );
+
+	describe( 'notifications', () => {
+		it( 'has appropriate notification methods', () => {
+			const actionsToExpect = [
+				'addNotification',
+				'removeNotification',
+				'fetchNotifications',
+				'receiveNotifications',
+				'receiveNotificationsFailed',
+			];
+			expect( Object.keys( actions ) ).toEqual( expect.arrayContaining( actionsToExpect ) );
```

Comment:
I know I'm guilty of suggesting this as a possible test when we spoke, but seeing it here I'm not sure it adds any value: this is essentially testing that the notifications API exports _at least_ these actions, even if our code never uses them. The notifications store is a public API for us and we should be sure to maintain it, but that's not what we should be testing here.

If the notifications API were to change we'd need to change this test… I'd rather a more "functional"/integration test that _used_ the notifications actions be included here so we'd be alerted to breaking changes in the notifications API.

-----

I'm sort of having a back-and-forth here and posting my thoughts, because ultimately I guess it's good to test this and make sure the module is behaving as expected. So we can leave this here, but I'm leaving the above waffling in case it's useful to have on record 😆 

Since there's a `notifications.js` file, though, these tests should go into a `notifications.test.js` file. I know it's a pain/fair bit of extra boilerplate, but we should keep tests as close to what they're testing as possible 😄